examples/apps/collaborative-textarea/package.json

```diff
@@ -53,7 +53,7 @@
   "devDependencies": {
     "@fluid-tools/webpack-fluid-loader": ">=2.0.0-internal.1.2.0 <2.0.0-internal.2.0.0",
     "@fluidframework/build-common": "^1.0.0",
-    "@fluidframework/eslint-config-fluid": "^0.28.2000",
+    "@fluidframework/eslint-config-fluid": "^1.0.0",
     "@fluidframework/test-tools": "^0.2.3074",
     "@fluidframework/test-utils": ">=2.0.0-internal.1.2.0 <2.0.0-internal.2.0.0",
     "@rushstack/eslint-config": "^2.5.1",
```

Comment:
Nit: is this still needed?

Reply:
Sorry, can you elaborate?

Reply:
Oh, github was highlighting the wrong line. No, this shouldn't be needed. Will remove.

Reply:
I also just put up a PR on main that removes this dep globally: #12144

Reply:
Hmm. Actually, in most packages this seems to cause problems. The config itself isn't needed, but the plugins that normally get pulled in with it are. I suspect we could work around this by using the inline require pattern in our base config, but that would need to be validated.